packages/chakra-components/src/MenuBar/SubMenu.js

```diff
@@ -67,34 +64,14 @@ const SubMenu = ({
       updateTabIndex(activeIndex);
     }
 
-    if (trigger === "click") {
-      if (activeIndex === -1 && !_isOpen && wasPreviouslyOpen) {
-        titleRef.current && titleRef.current.focus();
-      }
-    }
-
-    if (trigger === "hover") {
-      if (
-        activeIndex === -1 &&
-        !_isOpen &&
-        (wasPreviouslyOpenBeforeTimeout || wasPreviouslyOpen)
-      ) {
-        titleRef.current && titleRef.current.focus();
-      }
```

Comment:
Why was this removed?

Reply:
These are critical for rendering the focus back to the submenu after blur or mouse leave.
This set of code is necessary.

Reply:
rebase with refractor-menubar

Reply:
Done 👍